saleor/graphql/order/types.py

```diff
@@ -817,12 +820,30 @@ def resolve_number(root: models.Order, _info):
         return str(root.pk)
 
     @staticmethod
-    def resolve_payment_status(root: models.Order, _info):
-        return root.get_payment_status()
```

Comment:
Should we drop `get_payment_status`?

Reply:
I think so. It's not used anywhere now so let's get rid of it.